src/client/src/tscParseLineProtocol.c

```diff
@@ -1061,7 +1061,7 @@ static int32_t insertChildTablePointsBatch(TAOS* taos, char* cTableName, char* s
   tscDebug("SML:0x%"PRIx64" insert child table table %s of super table %s : %s", info->id, cTableName, sTableName, sql);
 
   size_t rows = taosArrayGetSize(rowsBind);
-  size_t maxBatchSize = TSDB_MAX_WAL_SIZE/rowSize * 4 / 5;
+  size_t maxBatchSize = TSDB_MAX_WAL_SIZE/rowSize * 2 / 3;
```

Comment:
it will be better if it can be changed by dynamic. Otherwise it may be happen again if input data change.